sqs5.tf

```
@@ -0,0 +1,18 @@
+locals {
+  sqs5_name = length(var.sqs5_name_override) > 0 ? var.sqs5_name_override : join("-", [var.name, "1"])
```

Shouldn't it be like so (`5` at the very end)?
```suggestion
  sqs5_name = length(var.sqs5_name_override) > 0 ? var.sqs5_name_override : join("-", [var.name, "5"])
```

---

mr eagle eye joined the game?
you´re correct but we also could keep it to force the users to create custom names for the sqs, in the other case they will delete the sqs after creating with  1,1,1,1,1 :D